com.github.devalien.workspaces.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "com.github.devalien.workspaces",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "command": "com.github.devalien.workspaces",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.Flatpak"
```

Comment:
Just to double check, this is to run `flatpak-spawn --host`?

Reply:
as above